pages/common/emacsclient.md

```diff
@@ -0,0 +1,11 @@
+# emacsclient
+
+> Open files in an existing emacs server.
+
+- Open file in console mode (without X window):
```

Comment:
You mention console mode here, but hint at a graphical interface. Perhaps an example demonstrating the graphical interface might be worth including here?